library/psa_crypto_storage.c

```diff
@@ -18,12 +18,6 @@
  *  limitations under the License.
  */
 
-#if defined(MBEDTLS_CONFIG_FILE)
```

Comment:
And you need to move the inclusion of common.h here as otherwise config.h is included too late.

Reply:
Oh I see I missed that